plugins/generator-1.15.2/forge-1.15.2/mappings/types.yaml

```diff
@@ -5,7 +5,7 @@ entity: Entity
 itemstack: ItemStack
 map: HashMap
 string: String
-boolean: boolean

```

Comment:
Check procedure dependencies, they use "boolean" right now so you need to change it there too, and on other places where it may need replacement now (eg. ParticleGUI class, Dependency class). Please test these changes well too.

Reply:
Oh, I thought I already changed procedure blocks. I will check that.